keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1179,17 +1179,9 @@ don't necessarily know what that is ahead of time.) Admins can use
 
 #### Behavior of service IPs
 
-```
-<<[UNRESOLVED unused service IP ports ]>>
-
-@thockin has suggested that service IPs should reject connections on
-ports they aren't using. (This would most easily be implemented by
-adding a `--service-cidr` flag to kube-proxy so we could just "reject
-everything else", but even without that we could at least reject
-connections on inactive ports of active service IPs.)
-
-<<[/UNRESOLVED]>>
-```
+Traffic to invalid [ports](https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports) on active cluster IPs will be rejected by 
+the nftables proxy. If the [MultipleServiceCIDR](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1880-multiple-service-cidrs) feature gate is enabled,
+it will additionally drop traffic to unassigned cluster IPs.
```

Comment:
KEP readers are assumed to be generally clueful about k8s, so you don't have to link to the end user docs for basic concepts.

The link to the ServiceCIDR KEP is good though, since that's not a well-established part of k8s yet. But I think using "endnote-style" links rather than inline links is part of the KEP style... or at least, the KEP template does it that way so I always just copied that in actual KEPs.

Also, the rest of the KEP is line-wrapped much shorter than that.

```suggestion
Traffic to invalid ports on active cluster IPs will be rejected by the nftables proxy.
If the [MultiServiceCIDRAllocator] feature gate is enabled, it will additionally drop
traffic to unassigned cluster IPs.

[MultiServiceCIDRAllocator]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1880-multiple-service-cidrs
```